social-network/src/PromoBox/BookmarkIcon.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+
+class bookmarkIcon extends Component {
+
+// const bookmarkIcon = React.createClass({
```

Comment:
For context, `React.createClass` has been removed completely from the main React library. There may be some examples out there still that use this method though.